design/one-pager-simple-class-selection.md

```diff
@@ -0,0 +1,411 @@
+# Simple Resource Class Selection
+* Owner: Nic Cope (@negz)
+* Reviewers: Crossplane Maintainers
+* Status: DRAFT
+
+## Background
+
+In Crossplane 0.3 we introduced support for "provider specific" (aka
+"non-portable", aka "[strongly typed]") resource classes. During the development
+cycle [we identified] that if a claim must explicitly reference a provider
+specific resource class in order to enable dynamic provisioning, then the claim
+itself is now provider specific and no longer portable across providers. We
+solved this issue by introducing [portable resource classes].
+
+A portable resource class is effectively an indirection to a non-portable
+resource class. Portable resource classes can be set as the default for a
```

Comment:
does this proposal also eliminate per namespace defaults, such that only cluster-scoped defaults will be available going forward?